policyengine_us/parameters/gov/states/vt/tax/income/credits/cdcc/low_income/rate.yaml

```
@@ -0,0 +1,11 @@
+description: Vermont matches this percent of the federal capped child and dependent care credit for low-income filers. 
```

```suggestion
description: Vermont matches this percent of the federal child and dependent care credit for low-income filers. 
```

Please add 2020 law to check if it's capped. In 2021 only (due to ARPA), the federal CDCC was refundable, so capping had no effect. The 2021 form refers to [2021 Form 2441](https://www.irs.gov/pub/irs-prior/f2441--2021.pdf) line 10, which is just the full CDCC.

Given the 2022 form did cap at tax liability, and the law looks the same, I suspect it was also capped. But we don't usually describe it as such so we can remove it either way.

---

yes, in the year 2020, its also limited by tax liability: [https://www.irs.gov/pub/irs-prior/f2441--2020.pdf#page1](url) line 11

---

For the tax years 2022 and 2020 the Vermont CDCC references the line 11 from [form 2441](https://www.irs.gov/pub/irs-pdf/f2441.pdf) - which is the capped cdcc we believe

In 2021 the CDCC refers to line 10 from the 2021 [Form 2441](https://www.irs.gov/pub/irs-prior/f2441--2021.pdf) - which is the refundable cdcc 

So in 2020 we should use capped_cdcc and in 2021 the cdcc.py 

2022 has a separate calculation with the capped_cdcc

---

Vermont didn't change its law in 2021 right? This is just the result of the federal CDCC being fully refundable in 2021 only (due to ARPA). So we should just use capped_cdcc in both cases. They'll be equal in 2021.